tutorials/single-page-tutorials/nested-nfts-with-wallet-lib.md

```diff
@@ -0,0 +1,423 @@
+# Create nested NFTs with the Wallet Library
+
+In this tutorial you will create three NFTs on the Shimmer testnet. On Shimmer, NFTs are not just dumb tokens, instead they have their own unique addresses and are able to own other NFTs. This concept is known as nested NFTs. In order to illustrate this concept, you will follow along the storyline below.
+
+## Storyline
+
+Let's imagine a game with avatars and items, which all are represented as unique NFTs on the Shimmer network. In this example it allows an avatar to not only hold items in-game, but its avatar NFT to also own its unique item NFTs directly on-DLT, in order to represent the dependencies between in-game components on the Shimmer network.
+
+![Blue Warrior](./images/warrior-blue.gif)
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/).
+- This tutorial will build on the tutorial [Testnet Wallet Setup](https://wiki.iota.org/tutorials/wallet-setup). If you haven't completed it, please do so, and afterward, come back here, or make sure you have a prefunded wallet. You will use the same folder and development environment as in the mentioned prerequired tutorial, allowing you to reuse the Stronghold account with funds on it.
+
+## Set Up
+
+### Prepare Your Development Environment
+
+Navigate to the folder you created when completing the [Testnet Wallet Setup](https://wiki.iota.org/tutorials/wallet-setup):
+
+```bash
+cd wallet-setup
+```
+
+#### Install Packages
+
+Next to the existing dependencies of the wallet setup tutorial, you will only need to add two more packages by running the following commands:
+
+```bash
+npm install @iota/client
+npm install ipfs-core@0.10.8
+```
+
+Afterward, your `package.json` file should contain the following dependencies:
+
+```json {6-12}
+{
+  "name": "wallet-setup",
+  "version": "1.0.0",
+  "description": "",
+  "main": "create-mnemonic.js",
+  "dependencies": {
+    "@iota/client": "^3.0.0-alpha.7",
+    "@iota/wallet": "^2.0.2-alpha.21",
+    "bip39": "^3.0.4",
+    "dotenv": "^16.0.1",
+    "ipfs-core": "^0.15.4"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "Apache-2.0"
+}
+```
+
+---
```

Comment:
Referring to the previous comment, I'd set this up as a fresh project with its own dependencies.

Reply:
Maybe we can discuss this in our call today. In case we want to establish learning paths, IMO these kind of dependencies shouldn't be problematic. In case we want every tutorial to be independent you're right of course.